api/models/flaggedItem/notifyUtils.js

```diff
@@ -4,25 +4,27 @@ export async function notifyModeratorsPost (flaggedItem) {
   const post = await flaggedItem.getObject()
   const user = flaggedItem.relations.user
   const communities = await user.communitiesSharedWithPost(post)
-  return sendToCommunities(flaggedItem, communities)
+  const isPublic = post.attributes.is_public
+  return sendToCommunities(flaggedItem, communities, isPublic)
 }
 
 export async function notifyModeratorsComment (flaggedItem) {
   const comment = await flaggedItem.getObject()
   const post = comment.relations.post
   const user = flaggedItem.relations.user
   const communities = await user.communitiesSharedWithPost(post)
-  return sendToCommunities(flaggedItem, communities)
+  const isPublic = post.attributes.is_public
+  return sendToCommunities(flaggedItem, communities, isPublic)
 }
 
 export async function notifyModeratorsMember (flaggedItem) {
   const member = await flaggedItem.getObject()
   const user = flaggedItem.relations.user
   const communities = await user.communitiesSharedWithUser(member)
-  return sendToCommunities(flaggedItem, communities)
+  return sendToCommunities(flaggedItem, communities, isPublic = false)
```

Comment:
just pass in false, no need to set a variable. im suprised there's no lint complaint about this